tests/backends/test_dbmodel.py

```diff
@@ -807,7 +807,7 @@ def test_inconsistent_params_db_model():
     # Create and save ML model to filesystem
     model, inputs, outputs = create_tf_cnn()
     with pytest.raises(SSUnsupportedError) as ex:
-        db_model = DBModel(
+        _ = DBModel(
```

Comment:
I think Toast dislikes bothering with an unused "anonymous" variable. Why bother with `_ = ` at all if you don't need it?

Reply:
Right. Did it on all occurrences you pointed out.